src/libs/url_utils.js

```diff
@@ -72,3 +72,9 @@ export function updateQueryString(queriesObject) {
     ...queriesObject,
   });
 }
+
+export function isFromQwant() {
+  const params = parseQueryString(window.location.search);
+  const qwantClients = ['search-ia-local', 'search-ia-multi', 'search-ia-address'];
```

Comment:
Unless I am mistaken, the "back to Qwant" button should be visible for "IA Multi" only. So I would suggest to rename this function to something like `shouldShowBackToQwant()` ?

Moreover, `search-ia-maps-multi` would be more explicit in my opinion. And `search-ia-local` will be obsolete soon, so no reason to mention it here.

Reply:
The button should be displayed on a poi panel too from the ticket's description. I'm ok with the renaming though :)